docs/photoniq/fingerprinting/fingerprinting-metrics.md

```diff
@@ -0,0 +1,18 @@
+---
+sidebar_position: 40
+title: Fingerprinting Metrics
+---
+
+The tracking metrics below quantify the scale of Digital Fingerprinting and can help identify usage trends.
+
+## Accuracy Rate
+
+Percentage of visitors correctly identified.
+
+## Unique Visitor Count
+
+The total number of unique visitors identified by Digital Fingerprinting over a period.
+
+## Requests per Second
+
+The average number of fingerprinting requests served per second in a given minute.
```

Comment:
@joubert-macrometa this we will remove in the next release

Reply:
Please let us know when that happens so that we can update this documentation.

Reply:
You can remove those now.  They have been removed.